indra/cmake/00-Common.cmake

```diff
@@ -182,6 +182,10 @@ if (LINUX OR DARWIN)
 
   list(APPEND GCC_WARNINGS -Wno-reorder -Wno-non-virtual-dtor )
 
+  if(LINUX)
+    list(APPEND GCC_WARNINGS -Wno-maybe-uninitialized -Wno-misleading-indentation -Wno-stringop-truncation -Wno-unused-value )
```

Comment:
Most of those are not needed and will be fixed in PR 1099

Reply:
thank you!